CONTRIBUTING.md

```diff
@@ -1,16 +1,30 @@
 # mongodb-operator
 
-## Code overview
 
-TODO: expand on this section when spec doc is created
+## Developing
 
-## Intended use case
+As this is a machine charm, it can be deployed locally using [Juju on a local LXD cloud](https://juju.is/docs/olm/lxd). Once the controller is created, to deploy this workload you need to follow some preliminar steps.
```

Comment:
Definitely, I **loved** that you did this, I wanted to change it from the Passive Voice to the Active Voice, since I think it makes things more clear. This is a nit :)